runtime/src/main/java/com/dylibso/chicory/runtime/MachineUtil.java

```diff
@@ -0,0 +1,89 @@
+package com.dylibso.chicory.runtime;
+
+import com.dylibso.chicory.wasm.exceptions.ChicoryException;
+import com.dylibso.chicory.wasm.types.Instruction;
+import com.dylibso.chicory.wasm.types.Value;
+import com.dylibso.chicory.wasm.types.ValueType;
+import java.util.Arrays;
+import java.util.List;
+
+public class MachineUtil {
```

Comment:
I'm not great with names, but wdyt about `MachineConstants` or `CompileTimeConstants`(and change the methods to e.g. `computeValue`), just to show the purpose of the class instead of using "Util".